New blog post: Bridging Learning Gaps Through Open-Source Tools by David Kiragu Mwaura (#139)
OBF/OBF.github.io: content/posts/2026-02-02-David-Mwaura-AIBBC2025.md

---
author: "David-Kiragu-Mwaura"
date: 2026-02-02
draft: true
category:
 - community
 - event-fellowship
 - travel-fellowship
tag:
 - community
 - event-fellowship
 - travel-fellowship

title: "Bridging Learning Gaps Through Open-Source Tools"
url: /2026/02/02/2026-02-02-David-Mwaura-AIBBC2025/
---

**_The_** [**_Open Bioinformatics Foundation (OBF) Event Fellowship program_**](/travel-awards) **_aims to promote diverse participation at events promoting open-source bioinformatics software development and open science practices in the biological research community. David Kiragu Mwaura,_** _**an Assistant Research Scientist at the**_ _**Kenya Institute of Primate Research**_, **_was awarded an OBF Event Fellowship to attend_** **_[2025 AIBBC conference](https://www.aibbc-society.org/))_**.

> 

Reviewer: It's possible that this blank line between the first paragraph and the image will mean the image will not show up on https://www.open-bio.org/blog/ - I'm not sure. But we can see and change it if needed.

Standing proudly beside my poster, I highlighted the complexity of teaching kinship inference and presented our solution, [DadApp](https://github.com/KIRAGU-MWAURA/DadApp_Shiny_Web_App), the open source R shiny tool. I shared the successful results from the Kenya Institute of Primate Research (KIPRE) pilot study that described the increase of interest and confidence upon using DadApp as an active learning tool. The results were uniquely compelling to the audience which sparked the discussion among lecturers and researchers from many institutions across Africa, who recognized the immense value of such open source tools. They saw an accessible, validated tool that could be replicated and scaled here in Africa where technology is gradually accessible.

![Audience for the poster](/img/2026-02-02-image-kiragu01.jpg.png)

> However, what stood out was how diverse my audience was: Some asked what Open Source is and why it matters in the Biotechnology and Biomedical fields? Whereas some audience members were advanced to ask what license is the Shiny App released under and if it allows commercial use, modification and/or redistribution? The diverse audience validated the continuous need for sensitizing about the importance of Open Source tools and Open Science in general. One of the biggest highlights which was satisfying was to experience a Professor from one of the leading universities in Kenya appreciating the content of the poster and commenting that such Open Source tools are the future as they can help in enhancing flexibility in curriculum design. He further added that lecturers and those passionate in capacity development can tailor such tools to course goals especially in lessons that are mathematically and coding intensive.

Reviewer 2: ```suggestion
However, what stood out was how diverse my audience was: Some asked what Open Source is and why it matters in the Biotechnology and Biomedical fields? Whereas some audience members were advanced to ask what license is the Shiny App released under and if it allows commercial use, modification and/or redistribution? The diverse audience validated the continuous need for sensitizing about the importance of Open Source tools and Open Science in general. One of the most satisfying highlights was having a Professor from one of the leading universities in Kenya appreciating the content of the poster and commenting that such Open Source tools are the future as they can help in enhancing flexibility in curriculum design. He further added that lecturers and those passionate in capacity development can tailor such tools to course goals especially in lessons that are mathematically and coding intensive.
```
